lms/templates/instructor/instructor_dashboard_2/queries.html

```diff
@@ -0,0 +1,140 @@
+<%! from django.utils.translation import ugettext as _ %>
+<%page args="section_data"/>
+<%! from microsite_configuration import microsite %>
+
+<script type="text/template" id="email-list-widget-template">
+    <div class="wrapper-email-select">
+        <div class="email-selection-label">{{label}}</div>
+        <select class="single-email-selector">
+              <option> ${_("Loading problem list")} </option>
+        </select>
+    </div>
+</script>
+
+<div
+    class="email-lists-management"
+    data-query-endpoint="${ section_data['get_single_query']}"
+    data-total-endpoint="${ section_data['get_all_students']}"
+    data-temp-queries-endpoint="${ section_data['get_temp_queries']}"
+    data-delete-batch-temp-endpoint="${ section_data['delete_temp_query_batch']}"
+    data-delete-saved-endpoint="${ section_data['delete_saved_query']}"
+    data-delete-temp-endpoint="${ section_data['delete_temp_query']}"
+>
+    <div class="email-widget">
+    <div class="header">
+        <h2 class="title">${_("Email Distribution")}</h2>
+    </div>
```

Comment:
just say `<h2>${_("Email Distribution")}</h2>` and get rid of the extraneous wrapper